.github/workflows/release.yml

```diff
@@ -47,17 +47,17 @@ jobs:
 
       # Merge @latest release back to @next.
       #
-      - name: Sync to next
-        if: "github.ref == 'refs/heads/main'"
-        uses: ad-m/github-push-action@master
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          branch: next
-        continue-on-error: true
+      #- name: Sync to next
+      #if: "github.ref == 'refs/heads/main'"
+      #uses: ad-m/github-push-action@master
+      #with:
+      #github_token: ${{ secrets.GITHUB_TOKEN }}
+      #branch: ${{ github.ref }}
+      #continue-on-error: true
```

Comment:
is losing the indenting an issue?

Reply:
Oh, I don't know why my editor does that.

Reply:
i mean it probably doesn't matter since it's commented out, heh